.github/dependabot.yml

```diff
@@ -1,18 +1,38 @@
 version: 2
 updates:
-  - package-ecosystem: "pip"
-    directory: "/"
-    schedule:
-      interval: "daily"
-
-  # Enable version updates for Docker
-  - package-ecosystem: 'docker'
-    directory: '/'
-    schedule:
-      interval: 'daily'
-      
-  - package-ecosystem: "github-actions"
-    directory: "/"
-    schedule:
-      # Check for updates to GitHub Actions every week
-      interval: "weekly"
\ No newline at end of file
+- package-ecosystem: pip
+  directory: "/"
+  schedule:
+    interval: weekly
+    day: wednesday
+    time: "03:00"
+    timezone: Europe/London
+  groups:
+    bundler:
```

Comment:
i think this is just a name we give it, perhaps `pip:`  inline with the ecosystem

Reply:
D'oh!